regrippy/plugins/usersids.py

```diff
@@ -10,9 +10,6 @@ class Plugin(BasePlugin):
     def run(self):
         path = r"Microsoft\Windows NT\CurrentVersion\ProfileList"
         key = self.open_key(path)
-        if not key:
-            print('Key not found')
-            return
 
```

Comment:
Sorry if my comment wasn't clear enough, but the error handling logic wasn't useless, only the `print()` statement was. If the key isn't found, `BasePlugin.open_key()` will return `None`, which has to be handled.